packages/client/src/components/app/filter/FilterPopover.svelte

```diff
@@ -33,6 +33,11 @@
   import dayjs from "dayjs"
   import utc from "dayjs/plugin/utc"
 
+  interface $$Slots {
```

Comment:
Why does this interface begin with `$$` ?

Reply:
That's just how you access the internal slots in Svelte 4. we needed to provide an interface to satisfy some new Svelte 5 compiler warnings.